ballerina/Ballerina.toml

```diff
@@ -5,6 +5,7 @@ version = "2.2.0"
 authors = ["Ballerina"]
 keywords = ["xslt", "xml", "html", "xsl", "transformation"]
 repository = "https://github.com/ballerina-platform/module-ballerina-xslt"
+icon = "resources/icon.png"
```

Comment:
Let's move icon to ballerina root directory

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-xslt/pull/368/commits/d6453f5181efdc30d0189285750ebd2b02be076d